book_source/developers_guide/Logging.Rmd

```diff
@@ -72,15 +72,15 @@ setLevel('INFO', getLogger('PEcAn.SQL'))
 ```
 
 To now use logging in the code you can use the following code:
-```r
+```{r loggingexample1, echo=TRUE}
 pl <- getLogger('PEcAn.MetaAnalysis.function1')
 pl$info("This is an INFO message.")
-pl$debug("The value for x=%d', x)
+pl$debug("The value for x=%d", x)
 pl$error("Something bad happened and I am scared now.")
 ```
 or
-```r
-loginfo("This is an INFO message.", logger='PEcAn.MetaAnalysis.function1')
-logdebug("The value for x=%d', x, logger='PEcAn.MetaAnalysis.function1')
-logerror("Something bad happened and I am scared now.", logger='PEcAn.MetaAnalysis.function1')
+```{r loggingexample1, echo=TRUE}
+loginfo("This is an INFO message.", logger="PEcAn.MetaAnalysis.function1")
+logdebug("The value for x=%d", x, logger="PEcAn.MetaAnalysis.function1")
```

Comment:
Since this code actually has to run, you'll want to make sure `x` is actually defined.